docs/faker/batch.md

```diff
@@ -0,0 +1,73 @@
+## Batch fake generator
+
+### Class
+
+* [BatchFakeGenerator](../../devoutils/faker/generators/batch_fake_generator.py)
+
+### Doc
+
+Sometimes we want to test a new application but we only have one
+small file with some sample events. In this case we want to be able
+to generate much more data from the original sample.
+
+The batch mode allows to generate events between two arbitrary dates as fast 
+as possible. The events will be dumped to the standard output and then you can
+redirect it to a file or somewhere else
+
+First we want to take the sample events and create a template from them. For
+example, if we have these sample events:
+
+```
+<14>2018-03-22T23:56:09.237-07:00 ABCDEFGHI01 firewall.cisco_asa: 2018-03-22T23:56:09.237067-07:00 : %ASA-123456: Teardown dynamic UDP translation from inside:123.123.123.123/00001 to outside:111.111.111.111/11111 duration 0:02:32
```

Comment:
The batch mode requires the events in raw format? if true we must put it in the documentation